samples/monitor-changes-to-map-load-status/src/main/java/com/esri/arcgismaps/sample/monitorchangestomaploadstatus/screens/MonitorChangesToMapLoadStatusScreen.kt

```diff
@@ -0,0 +1,94 @@
+/* Copyright 2025 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgismaps.sample.monitorchangestomaploadstatus.screens
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+import com.esri.arcgismaps.sample.monitorchangestomaploadstatus.components.MonitorChangesToMapLoadStatusViewModel
+import com.esri.arcgismaps.sample.sampleslib.components.MessageDialog
+import com.esri.arcgismaps.sample.sampleslib.components.SampleTopAppBar
+
+/**
+ * Main screen that displays a MapView and an overlay showing the map's load status.
+ */
+@Composable
+fun MonitorChangesToMapLoadStatusScreen(sampleName: String) {
+    val mapViewModel: MonitorChangesToMapLoadStatusViewModel = viewModel()
+
+    // Observe the load status from the ViewModel
+    val loadStatusText by mapViewModel.loadStatusText.collectAsStateWithLifecycle(
+        initialValue = "Not Loaded"
+    )
+
+    Column(modifier = Modifier.fillMaxSize()) {
+        SampleTopAppBar(title = sampleName)
+
+        Box(modifier = Modifier
+            .fillMaxSize()
+            .padding(top = 0.dp)) {
```

Comment:
To keep samples consistent here, I would recommend using a scaffold with the inner padding: 
 
```suggestion
    Scaffold(topBar = { SampleTopAppBar(title = sampleName) }) { padding ->

        Box(modifier = Modifier
            .fillMaxSize()
            .padding(padding)) {
```